packages/server/lib/controllers/auth/tba.ts

```diff
@@ -1,30 +1,30 @@
-import axios from 'axios';
-import querystring from 'querystring';
-import * as uuid from 'uuid';
 import { z } from 'zod';
-import * as crypto from 'node:crypto';
-import type { OAuthSession } from '@nangohq/shared';
+import tracer from 'dd-trace';
 import { asyncWrapper } from '../../utils/asyncWrapper.js';
-import { zodErrorToHTTP, generateBaseString, generateSignature, getTbaMetaParams, SIGNATURE_METHOD, percentEncode } from '@nangohq/utils';
-import { analytics, configService, AnalyticsTypes, getConnectionConfig, getOauthCallbackUrl, interpolateStringFromObject } from '@nangohq/shared';
-import oAuthSessionService from '../../services/oauth-session.service.js';
-import { missesInterpolationParam } from '../../utils/utils.js';
-import * as WSErrBuilder from '../../utils/web-socket-error.js';
-import type { TbaAuthorization } from '@nangohq/types';
+import { zodErrorToHTTP } from '@nangohq/utils';
+import { analytics, configService, AnalyticsTypes, getConnectionConfig, connectionService } from '@nangohq/shared';
+import type { TbaAuthorization, TbaCredentials } from '@nangohq/types';
 import { defaultOperationExpiration, logContextGetter } from '@nangohq/logs';
 import { hmacCheck } from '../../utils/hmac.js';
+import { connectionCreated as connectionCreatedHook, connectionTest as connectionTestHook } from '../../hooks/hooks.js';
+
+const bodyValidation = z
+    .object({
+        token_id: z.string().nonempty(),
```

Comment:
string nonempty is deprecated I think. `.min(1)` should be used

Reply:
Updated